monet/util/stats.py

```diff
@@ -1774,39 +1773,63 @@ def CSI(obs, mod, minval, maxval):
 
 
 def scores(obs, mod, minval, maxval=1.0e5):
-    """Short summary.
+    """Calculate scores.
 
     Parameters
     ----------
-    obs : type
-        Description of parameter `obs`.
-    mod : type
-        Description of parameter `mod`.
-    minval : type
-        Description of parameter `minval`.
-    maxval : type
-        Description of parameter `maxval`.
+    obs : array-like
+        Observation values ("truth").
+    mod : array-like
+        Model values ("prediction").
+        Should be the same size as `obs`.
+    minval, minval : float
+        Interval to test (exclusive on both sides).
 
     Returns
     -------
-    type
-        Description of returned object.
-
+    a, b, c, d : float
+        Counts of hits, misses, false alarms, and correct negatives.
     """
-    d = {}
-    d["obs"] = obs
-    d["mod"] = mod
-    df = DataFrame(d)
-    ct = crosstab(
+    import pandas as pd
+
+    df = pd.DataFrame({"obs": obs, "mod": mod})
+
+    # If NaN is involved in a cond, it will be F, we want to skip those
+    df = df.dropna(subset=["obs", "mod"], how="any")
+
+    if df.empty:
+        return 0, 0, 0, 0
```

Comment:
```suggestion
        zero = np.float64(0)
        return zero, zero, zero, zero
```
For consistency with the normal return (and the docstring says float)